lib/gst/player/gstplayer.c

```diff
@@ -52,6 +52,8 @@
 GST_DEBUG_CATEGORY_STATIC (gst_player_debug);
 #define GST_CAT_DEFAULT gst_player_debug
 
+#define DEFAULT_POSITION_UPDATE_INTERVAL_MS 100
```

Comment:
Now only add a commit to add #defines for the other properties, and then I can merge :) Thanks!